Bottom: 34dbf5d699e1729ef0503706bba487cee34954a7
Top:    6045a4ac6b9a974e6e696884e398a68495d14ba6
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 14:44:29 -0500

Refresh of known_hosts

---
diff --git a/ansible-playbook/buildbot.yml b/ansible-playbook/buildbot.yml
index 148ed7aa..6a605b17 100644
--- a/ansible-playbook/buildbot.yml
+++ b/ansible-playbook/buildbot.yml
@@ -36,7 +36,7 @@
     - buildbot-worker
   vars:
     buildbot_master: "{{ hostvars['buildbot-master'].ip_addr }}"
-    known_hosts: "{{gitlab_keys.stdout}}"
+    known_hosts: "{{ gitlab_keys.stdout }}"
   tasks:
     - name: Scan Gitlab ssh keys
       command: ssh-keyscan {{gitlab_hostname}}
